.prettierrc

```diff
@@ -0,0 +1,23 @@
+{
+	"printWidth": 110,
+	"useTabs": true,
+	"semi": true,
+	"singleQuote": true,
+	"trailingComma": "none",
+	"bracketSpacing": false,
+	"arrowParens": "avoid",
+	"endOfLine": "lf",
+	"quoteProps": "consistent",
+	"overrides": [
+		{
+			"files": "*.md",
+			"options": {
+				"printWidth": 70,
+				"useTabs": false,
+				"trailingComma": "none",
+				"arrowParens": "avoid",
+				"proseWrap": "never"
```

Comment:
Likewise here. If you're able to justify the overwrites then we'll stick with them, otherwise let's let prettier handle the rules for us

Reply:
`.prettierrc` file is like json so json rules are applied. Including TABS